etc/scipipe/verify_drp_metrics.yaml

```diff
@@ -15,8 +15,8 @@ template:
       clone_timelimit: 15
     hsc: &dataset_hsc
       display_name: hsc
-      name: validation_data_hsc
-      github_repo: lsst/validation_data_hsc
+      name: rc2_subset
+      github_repo: lsst-dm/rc2_subset
```

Comment:
All test datasets seem to be in github.com/lsst . Why is this one going in githib.com/lsst-dm?

Reply:
Purely trying to avoid red tape while getting this change made.  If I can unilaterally move it, I will

Reply:
This was decided on the ticket.  We will move it, but not until a later date, so we will merge for now with the repo in its current location.